feeds/packagist/packagist_test.go

```diff
@@ -40,19 +43,60 @@ func TestFetch(t *testing.T) {
 func changesMock(w http.ResponseWriter, r *http.Request) {
 	if r.URL.Query().Get("since") == "" {
 		w.WriteHeader(http.StatusBadRequest)
-		w.Write([]byte(`{"error":"Invalid or missing \u0022since\u0022 query parameter, make sure you store the timestamp at the initial point you started mirroring, then send that to begin receiving changes, e.g. https:\/\/packagist.org\/metadata\/changes.json?since=16145146222715 for example.","timestamp":16145146222715}`))
+		_, err := w.Write([]byte(`{"error":"Invalid or missing \u0022since\u0022 query parameter,
+		make sure you store the timestamp at the initial point you started mirroring,
+		then send that to begin receiving changes,
+		e.g. https:\/\/packagist.org\/metadata\/changes.json?since=16145146222715 for example."
+		,"timestamp":16145146222715}`))
+		if err != nil {
+			fmt.Println("Unexpected error during mock http server write: %w", err)
+		}
+	}
+	_, err := w.Write([]byte(`{"actions":[{"type":"delete","package":"to-delete/deleted-package",
+	"time":1614513806},{"type":"update","package":"ossf/package","time":1614514502},
+	{"type":"update","package":"ossf/package~dev","time":1614514502}],"timestamp":16145145025048}`))
+	if err != nil {
+		fmt.Println("Unexpected error during mock http server write: %w", err)
```

Comment:
In a test this should raise the HTTP error similarly to the implementation of HttpServe outside of the tests (duplicate)